src/client/pages/Profile.js

```diff
@@ -0,0 +1,149 @@
+import React, { useEffect, useState } from 'react';
+import { useHistory, useLocation } from 'react-router-dom';
+import Navbar from '../components/Navbar';
+import useFetch from '../hooks/useFetch';
+
+import Button from '../components/Button';
+
+import '../style/Profile.css';
+
+function Profile() {
+  const location = useLocation();
+  const history = useHistory();
+  const [username, setUsername] = useState();
+  const [email, setEmail] = useState();
+  const [about, setAbout] = useState();
+  const [name, setName] = useState();
+  const [loc, setLocation] = useState();
+  const [verified, setVerified] = useState();
+  const [DOB, setDOB] = useState();
+  const [ID, setID] = useState(2);
+
+  const endPoint = `/api/getProfile/${ID}`;
+  const [isLoading, data, error] = useFetch(endPoint);
+  const dataList = data || [];
+
+
+  useEffect(() => {
+    setUsername(dataList.Username);
```

Comment:
Please use lower case for object properties. Those may be from the backend, then looking into backend.

Reply:
which part of the code is object property? Is it "setUsername" or "dataList" or "Username" or all of them?